build.assets/Dockerfile-teleterm

```diff
@@ -15,7 +15,7 @@ RUN BIN="/usr/local/bin" && \
     chmod +x "${BIN}/${BINARY_NAME}"
 
 # Install node
-ARG NODE_VERSION=v15.14.0
+ARG NODE_VERSION=v16.14.2
```

Comment:
This change is irrelevant to the rest of the changes, right? Drone won't be building this inside Docker?

Which makes me wonder: how are we going to ensure that Drone has the correct Node version?

Reply:
Correct, but our README says node16 so I figured I would change this while I'm at it.

Drone will build the linux builds in docker when we get there.

For the mac builders, the node version is set in a separate PR: https://github.com/gravitational/ops/pull/355

Reply:
It's now set in the `build.assets` makefile, with all the other tool versions

Reply:
...and passed in from the `build.assets` makefile